README.md

```diff
@@ -2,6 +2,42 @@
 
 > If you are using GitPod for the project exercise (i.e. you cannot use your local machine) then you'll want to launch a VM using the [following link](https://gitpod.io/#https://github.com/CorndelWithSoftwire/DevOps-Course-Starter). Note this VM comes pre-setup with Python & Poetry pre-installed.
 
+# Getting started for Project Exercise 2
+
+For universal access to our To Do items - we will be saving our Items in Trello using REST API calls. To get started:
+
+1. Register for a if you do not have one already [Trello account](https://trello.com/signup)
+
+2. Generate an API Key and Token [here](https://trello.com/app-key)
+
+3. Add Handy PowerUp App [API Developer ID Helper](https://trello.com/power-ups) to help with extracting board and List IDs.
+
+4. Save these credentials and IDs by updating the`.env` file from exercise 1:
+
+   ```
+   # Trello Api Management
+   TRELLO_API_KEY=<FILL ME>
+   TRELLO_TOKEN=<FILL ME>
+   BOARD_ID=<FILL ME>
+   TRELLO_DONE=<FILL ME>
+   TRELLO_IN_PROGRESS=<FILL ME>
+   TRELLO_TO_DO=<FILL ME>
+   BASE_URL='https://api.trello.com/1/'
+
+   ```
```

Comment:
This is really helpful, and could be even more so if we pasted this directly into the `.env.template` file!

Reply:
Thanks Jack... I will carry over the changes to Module 3 and merge this for now.